### ISSUE TITLE
Admin Page Error

### ISSUE THREAD

**Issue by Waqass** [dhtml/flarum-ext-preview-discussion#1]
Once I installed this extension I am unable to navigate to the admin page. Posting and viewing forum is fine only that when I go to the https://lookouthost.com/admin it give the following error 
An error occurred while trying to load this page.

There is no output in error log though.

**Comment by QGrain** [comment]
Me to. Is there any solution?

**Comment by mattish91** [comment]
> Me to. Is there any solution?

Yes, trough ssh, go to your /public folder and run "composer remove anthonyogundipe/preview-discussion". Just found out the hard way, probably should have noticed that this is abandoned or at least not compatible any more.
